packages/event-bus-default/src/services/event-bus.ts

```diff
@@ -0,0 +1,92 @@
+import { ConfigModule, EventHandler, IEventBusService, Logger, MedusaContainer, TransactionBaseService } from "@medusajs/medusa"
+import { EntityManager } from "typeorm"
+
+type InjectedDependencies = {
+  logger: Logger
+}
+
+/**
+ * Can keep track of multiple subscribers to different events and run the
+ * subscribers when events happen. Events will run asynchronously.
+ */
+export default class DefaultEventBusService extends TransactionBaseService implements IEventBusService {
+  protected readonly container_: MedusaContainer & InjectedDependencies
+  protected readonly config_: ConfigModule
+  protected readonly logger_: Logger
+  protected readonly manager_: EntityManager
+  protected readonly transactionManager_: EntityManager | undefined
+
+  constructor(
+    container: MedusaContainer & InjectedDependencies,
+    manager,
+    config: ConfigModule
+  ) {
+    // eslint-disable-next-line prefer-rest-params
+    super(arguments[0])
+    
+    this.container_ = container
+    this.config_ = config
+    this.logger_ = container.logger
+    this.manager_ = manager
+  }
+
+  /**
+   * @return this
+   */
+  subscribe(event: string | symbol, handler: EventHandler): this {
+    this.logger_.info(
+      `[${String(event)}] No event bus installed. Subscribe is unavailable.`
+    )
+    return this
+  }
+
+  /**
+   * @return this
+   */
+  unsubscribe(event: string | symbol, subscriber: EventHandler): this {
+    this.logger_.info(
+      `[${String(event)}] No event bus installed. Unsubscribe is unavailable.`
```

Comment:
suggestion: event.toString() should suffice

Reply:
same for other places below

Reply:
Curious, what's the difference?

Reply:
The main diff is that if the value is null or undefined, .toString will throw an error whereas String will not

Reply:
<img width="1359" alt="Screenshot 2022-11-25 at 11 07 10" src="https://user-images.githubusercontent.com/25098370/203958083-74b8d2d9-9e6c-48fc-84db-ee89bec1dedf.png">

Reply:
<img width="792" alt="Screenshot 2022-11-25 at 11 10 58" src="https://user-images.githubusercontent.com/25098370/203958145-3a14058c-8317-43d8-ac1d-3fafd613d78a.png">

Reply:
In that case, would it not make sense to use `String(...)` to avoid errors on missing event names 🤔

Reply:
I would argue that this is something that should never happened and we should be aware if it happen